src/main/java/hudson/plugins/distfork/DistForkCommand.java

```diff
@@ -124,6 +124,8 @@ public void run() {
                 try {
                     Computer c = Computer.currentComputer();
                     Node n = c.getNode();
+                    String nodeName = n.getNodeName().equals("") ? "master" : n.getNodeName();
```

Comment:
no.  it is `@NonNull` [Javadoc](http://javadoc.jenkins-ci.org/hudson/model/Node.html#getNodeName())